src/index.css

```
@@ -150,10 +184,42 @@
 
   /* Cards — most prominent surface, heaviest rim */
   .glass-card {
-    @apply rounded-3xl bg-white/5 backdrop-blur-3xl backdrop-saturate-150 transition-all duration-300;
-
-    border: none;
-    box-shadow: var(--glass-rim-strong), var(--shadow-glass);
+    @apply rounded-3xl border border-white/5;
+    position: relative;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)